src/components/App.css

```diff
@@ -7,9 +7,30 @@ body{
 
   justify-content: center;
   align-items: center;
+
+  background-color: whitesmoke;
 }
 
 h1, h2, h3{
   text-align: center;
   color: #009aa8;
+}
+
+select,
+input {
+	width: stretch;
+  height: 25px;
+	padding: 5px 15px;
+	text-align: left;
+  border: 0px;
+  border-bottom: 1px solid #009aa8;
+  background-color: white;
+}
+
```

Comment:
We should focus on the alignment

Reply:
I don't really understand what you mean, could you elaborate?
At the moment I use width: stretch to fill the available width given to the input. As this is only the styling for the <input> element I don't want to give it outside alignments as it may mess with the layout.

Reply:
What I mean is the alignment of code, not what CSS is doing.

Reply:
Noted, I will pay more attention in the future.